Sources/MapboxNavigation/InstructionsCardView.swift

```diff
@@ -7,16 +7,14 @@ public class InstructionsCardView: BaseInstructionsBannerView {
     
     var step: RouteStep!
     var distanceFromCurrentLocation: CLLocationDistance!
-    var gradientLayer: CAGradientLayer!
```

Comment:
The original design of the guidance card layout emphasized the gradients, but it would make sense to change the design at this point to be flatter. My only concern is that, without a gradient, the card can often lack contrast with the surrounding map. In the default day style, a white, non-immediate card is barely discernible from the map with a white background. We could mitigate this issue in the same way as we’ve done with the banners: by applying a subtle outline around the view.

Reply:
Agree, added stylable border in latest commit. Some example:

![Simulator Screen Shot - iPhone 12 - 2022-01-28 at 15 00 18](https://user-images.githubusercontent.com/1496498/151621585-39a71d01-4a6d-4889-83ed-0f1a86082925.png)

![Simulator Screen Shot - iPhone 12 - 2022-01-28 at 15 02 46](https://user-images.githubusercontent.com/1496498/151621616-99ab7e9c-858e-4490-ba42-061497683e27.png)